packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -149,9 +151,52 @@ export function setValueForProperty(
   if (shouldIgnoreAttribute(name, propertyInfo, isCustomComponentTag)) {
     return;
   }
+
+  if (
+    enableCustomElementPropertySupport &&
+    isCustomComponentTag &&
+    name[0] === 'o' &&
+    name[1] === 'n'
+  ) {
+    let eventName = name.replace(/Capture$/, '');
+    const useCapture = name !== eventName;
+    const nameLower = eventName.toLowerCase();
+    if (nameLower in node) {
```

Comment:
Removing this `if` block doesn't fail any tests. What is it for? Do we need a test for coverage?

Reply:
I copied this over from these lines in preact: https://github.com/preactjs/preact/blob/dd1e281ddc6bf056aa6eaf5755b71112ef5011c5/src/diff/props.js#L88-L90

I suppose it's to make it so `onClick` listens for an event named `click` instead of `Click`, but only if the element has an `onclick` property. The comment suggests that this only matters for builtin elements, and since removing this line still makes the `<my-custom-element onClick={...} />` test pass, it can be removed. I'll remove it.